aspnetcore/migration/70-80.md

```diff
@@ -73,64 +73,541 @@ In the project file, update each [Microsoft.AspNetCore.*](https://www.nuget.org/
+   @rendermode RenderMode.InteractiveWebAssembly
+   ```
+
+   Such components only render on the client after the Blazor bundle has been downloaded and the Blazor runtime activates. Keep in mind that component code is ***not*** private using the interactive WebAssembly render mode.
+
+   For additional information on applying render modes to components, see <xref:blazor/components/render-modes>.
+
+1. Run the solution from the ***Blazor Server*** project:
+
+   For Visual Studio, confirm that the Blazor Server project is selected in **Solution Explorer** when running the app.
+   
+   If using the .NET CLI, run the project from the Blazor Server project's folder.
+
+   For the `Counter` component of the `.Client` project, navigate to `/counter`.
+
+   If you added the `Counter2` component, which is strictly configured for interactive server rendering, navigate to `/counter-2`.
+
+### Convert a Blazor Server app into a Blazor Web App
+
+To adopt all of the changes that were made to the project templates for this release, we recommend the following process:
+
+* Create a new project from the Blazor Web App project template. For more information, see <xref:blazor/tooling>.
+* Move the app's components and code to the new app, making modifications to adopt new features.
+* Update the layout and styles of the Blazor Web App to match the Blazor Server app.
+
+New .NET 8 features are covered in <xref:aspnetcore-8#blazor>. When updating an app from .NET 6 or earlier, see the migration and release notes (*What's new* articles) for intervening releases.
+
+### Update a Blazor WebAssembly app
+
+Follow the guidance in the first three sections of this article:
+
+* [Update .NET Core SDK version in global.json](#update-net-core-sdk-version-in-globaljson)
+* [Update the target framework](#update-the-target-framework)
+* [Update package references](#update-package-references)
+
+Adopt [.NET 8 features](xref:aspnetcore-8#blazor). When updating an app from .NET 6 or earlier, see the migration and release notes (*What's new* articles) for intervening releases.
+
+### Update a hosted Blazor WebAssembly app
+
+To upgrade an existing ASP.NET Core hosted Blazor WebAssembly app to .NET 8:
+
+1. Follow the guidance in the first three sections of this article:
+
+   * [Update .NET Core SDK version in global.json](#update-net-core-sdk-version-in-globaljson)
+   * [Update the target framework](#update-the-target-framework)
+   * [Update package references](#update-package-references)
+
+   > [!IMPORTANT]
+   > Using the preceding guidance, update the `Client`, `Server`, and `Shared` projects of the solution.
+
+   For information on .NET 8 features, see <xref:aspnetcore-8#blazor>. When updating an app from .NET 6 or earlier, see the migration and release notes (*What's new* articles) for intervening releases.
+
+1. Move the file content from the `Client/wwwroot/index.html` file to a new `Host` component file (`Host.razor`) created at the root of the `Client` project.
```

Comment:
Sorry, my Net8BlazorWebAssembly app is a bit outdated. We should follow the Blazor Web App patterns here of App.razor and Routes.razor. App.razor stays in the server project. Routes.razor goes in the client project.

```suggestion
1. Move the file content from the `Client/wwwroot/index.html` file to a new `App` component file (`App.razor`) created at the root of the `Server` project.
```